routes/users.js

```diff
@@ -215,6 +216,80 @@ router.get("/:userId/offerings/inactive", async (req, res) => {
   }
 });
 
+// Get all the offers made to active postings belonging to the user
+router.get("/:userId/offerings/queue", async (req, res) => {
+  let offerings = [];
+
+  try {
+    const postingsOfUser = await Posting.find({
+      ownerId: req.params.userId,
+      active: true,
+    });
+
+    postingsOfUser.forEach((posting) => {
```

Comment:
I think it's probably more efficient and better for scalability to use joins:
https://docs.mongodb.com/master/reference/operator/aggregation/lookup/#pipe._S_lookup